dotcom-rendering/src/web/components/ImageComponent.tsx

```diff
@@ -283,6 +307,7 @@ export const ImageComponent = ({
 	) {
 		return (
 			<div
+				id={element.position ? `img-${element.position}` : ''}
```

Comment:
This this needs to be `element.position !== undefined ? `img-${element.position}` : ''`

Reply:
Thanks

👍 [fixed](https://github.com/guardian/dotcom-rendering/pull/7129/commits/84cc2534c275cd54bbd8ce88ba6c3bb8093a9fdf)